integration-tests/.env

```diff
@@ -7,7 +7,7 @@ POSTGRES_PORT=5432
 POSTGRES_DB=registry
 HIVE_ENCRYPTION_SECRET=wowverysecuremuchsecret
 HIVE_EMAIL_FROM=no-reply@graphql-hive.com
-HIVE_APP_BASE_URL=http://localhost:8080
+HIVE_APP_BASE_URL=http://0.0.0.0:8080
```

Comment:
Note that 0.0.0.0 does not work on windows and in general 127.0.0.1 or localhost is recommended over 0.0.0.0, as 0.0.0.0 is not a real usable IP address

Reply:
but `localhost` does not work on locally for me

Reply:
try to remove `.env` files, run `pnpm local:setup`, `pnpm dev` for a few services and you will see.

Reply:
okay you convinced me, fuck windows